### ISSUE TITLE
PG RC6: Large Video shows wrong/missing info

### ISSUE THREAD

**Issue by Nostang3** [puregallery/Pure-Gallery-Support#88]
I have a 16.91 GB video in a folder and when you open it in media viewer or just long press to select then choose info, it will show it being only 981mb. All videos shows 0x0 as resolution as well.

**Comment by Nostang3** [comment]
@puregallery So the app says RC9 but all your tags say RC8. Are they the same or did you fix all the related issues in an unplublished build or did you name the build version in the app wrong? I ask because I don't see an RC9 Milestone.

**Comment by puregallery** [comment]
Ah, yes...found an issue in the RC8 release. So, had to push another one and bump the number. I will change the milestone.

**Comment by Nostang3** [comment]
In RC11 a 2gb video shows -1944673.26KB.

**Comment by puregallery** [comment]
I don't have large videos on my device...need some time to get them and test this

**Comment by puregallery** [comment]
Is this still a bug?

**Comment by Nostang3** [comment]
Yeah. I just checked. I put a 14GB mp4 on the device and it shows 

```
0x0
-1387979.35kb
```
